### PR TITLE
consolidate typo PRs and update docs on instance sharing

### PR DESCRIPTION
**Pull request by ChrisMcGowan**
## Changes proposed in this pull request:
- Typo fixes
- Add segment for instance sharing under deployment/managed services

:sunglasses:[PREVIEW URL](https://cg-88d42ca6-59d7-47e0-9500-4dd9251360b9.app.cloud.gov/preview/cloud-gov/cg-site/fixes_and_sharing)

Consolidated and closed PRs being done in this PR:
- https://github.com/cloud-gov/cg-site/pull/1819
- https://github.com/cloud-gov/cg-site/pull/1818
- https://github.com/cloud-gov/cg-site/pull/1817

## Security Considerations
n/a

### CODE REVIEW

File: _docs/deployment/managed-services.md [hunk 1]
```diff
@@ -35,9 +35,17 @@ Create a new service instance by specifying a service, plan, and a name of your
 % cf create-service SERVICE_NAME PLAN_NAME INSTANCE_NAME
 ```
 
+### Sharing service instances
+
+Sharing of service instances allows customers to share service instances between spaces inside their organization as a way to consolidate service instance usage and avoid data duplication in their service instances.  Customers need to be aware that sharing service instances between spaces has limitations and can expose data in those service instances across environment boundaries.  Please see [Sharing Service Instances](https://docs.cloudfoundry.org/devguide/services/sharing-instances.html) for more information on limitations and security concerns.
```
Comment (bengerman13) [inline _docs/deployment/managed-services.md]: Can we say "users" instead of "customers"?

**Comment by bengerman13** [comment]
Am I seeing this correctly, that we manually replicated the changes from these commits rather than cherry-picking them?

File: _docs/deployment/managed-services.md (hunk 1 above)
Comment (ChrisMcGowan) [reply _docs/deployment/managed-services.md]: Sure i'm fine with that @bengerman13

**Comment by ChrisMcGowan** [comment]
@bengerman13 yes I just manually made the commits - not sure how to do the cherry-pick process

**Comment by bengerman13** [comment]
Happy to walk you through it after standup if you have a minute. 
(It's also probably fine in this case, for very small commits to wording, but for anything of substance we should use cherry-picks to make sure authors are attributed and keep commits atomic)

**Review by bengerman13 — APPROVED** [review]

**Pull request merged**